plugins/clnrest/utilities/rpc_plugin.py

```diff
@@ -1,9 +1,12 @@
 import os
 from pyln.client import Plugin
+from pathlib import Path
 
 plugin = Plugin(autopatch=False)
 
-plugin.add_option(name="rest-certs", default=os.getcwd(), description="Path for certificates (for https)", opt_type="string", deprecated=False)
+rest_certs = Path(os.getcwd()) / 'clnrest'
+
+plugin.add_option(name="rest-certs", default=rest_certs.as_posix(), description="Path for certificates (for https)", opt_type="string", deprecated=False)
```

Comment:
I think that this dependency from cln-grpc plugin certificates was a "feature" but I am not sure why we should not depend from them?

Reply:
The failing test that makes me do that change is `test_clnrest_does_not_depend_on_grpc_plugin_certificates`.

This is related to the fact that `cln-grpc` doesn't add `rest-host` IP in Subject Alternative Name of the certificates.  So when you start both `cln-grpc` and `clnrest` by specifiying a value for `rest-port` (let say `3010`) at startup and you try to do a `GET https://127.0.0.1:3010/v1/list-methods` request with the `ca.pem` generated by `cln-grpc`, you get a tls/ssl error.

As I mentioned in "Differences with previous versions" doc section, if someone wants to use the same certificates as `cln-grpc` uses, they can set `rest-certs` option accordingly.

Is is clearer or am I missing something?